test/functional/full_text_search/search_controller_test.rb

```diff
@@ -215,13 +215,18 @@ def format_api_results(items, total_count: nil)
           datetime ||= changeset.committed_on
         end
         datetime ||= item.updated_on if item.respond_to?(:updated_on)
-        datetime ||= item.created_on if item.respond_to?(:created_on)
+        if item.respond_to?(:created_on)
+          datetime ||= item.created_on
+          registered_at = item.created_on
+        end
```

Comment:
This will simplify the following code (`registered_at&.iso8601 || datetime&.iso8601` -> `registered_at*.iso8601`):

```suggestion
        else
          registered_at = datetime
        end
```

Reply:
fix: [f9fbeb0](https://github.com/clear-code/redmine_full_text_search/pull/119/commits/f9fbeb0ecaf5017cd84078e047ed5d517750deaf)